packages/appstore/src/components/CFDs/cfd-accounts.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import CFDDemoAccounts from './cfd-demo-accounts';
+import CFDRealAccounts from './cfd-real-accounts';
+import { isLandingCompanyEnabled, CFD_PLATFORMS, isMobile } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { Text, StaticUrl } from '@deriv/components';
+import { TPlatform, TAccountCategory, TMt5StatusServer, TMt5StatusServerType, TRootStore } from 'Types';
+import { useStores } from 'Stores/index';
+import { CompareAccountsModal } from '@deriv/cfd';
+
+type TCFDAccountsProps = {
+    account_type: TAccountCategory;
+};
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+const CFDAccounts = ({ account_type }: TCFDAccountsProps) => {
+    const store = useStores();
+    const { client, modules }: TRootStore = store;
+    const {
+        is_eu,
+        is_eu_country,
+        landing_companies,
+        is_logged_in,
+        is_populating_mt5_account_list,
+        is_populating_dxtrade_account_list,
+        website_status,
+        mt5_disabled_signup_types,
+        dxtrade_disabled_signup_types,
+        dxtrade_accounts_list_error,
+        has_active_real_account,
+        trading_platform_available_accounts,
+    } = client;
+
+    const {
+        current_list,
+        is_compare_accounts_visible,
+        toggleCompareAccountsModal,
+        setAccountType,
+        enableCFDPasswordModal,
+    } = modules.cfd;
+    const is_demo_tab = account_type === 'demo';
+
+    const openRealPasswordModal = (account_creation_account_type: TOpenAccountTransferMeta) => {
+        setAccountType(account_creation_account_type);
+        enableCFDPasswordModal();
+    };
+
+    const hasAccount = (platform: TPlatform, landing_company_short?: string) =>
+        Object.keys(current_list).some(key =>
+            landing_company_short
+                ? key.startsWith(`${platform}.${account_type}`)
+                : key.startsWith(`${platform}.${account_type}.${landing_company_short}`)
+        );
+
+    const isDerivedVisible = (platform: TPlatform) => {
+        // Hiding card for logged out EU users
+        if (!is_logged_in && is_eu_country) return false;
+
+        if (is_eu && hasAccount(platform, 'synthetic')) return false;
```

Comment:
```suggestion
        if (!is_logged_in && is_eu_country) || (is_eu && hasAccount(platform, 'synthetic')) return false;
```
This can be combined since if either one if these two conditions is true this function should return false